File: credit/credit.c
#include <cs50.h>
#include <stdio.h>

int sumDigits(int n)
{

}

int main(void)
{
    long num = get_long("Number: ");
    int i = 2;
    int multiplied = 0;
    int notMultiplied = 0;
    int noOfDigits = 0;

    while(true)
    {
         if(num<10)
         {
             if(i % 2 == 1)
             {
                num = num * 2;
                if(num > 10)
                {
                    num = sumDigits(num);
                }
                multiplied += num;
             }
            else if(i % 2 == 0)
            {
                notMultiplied += num;
            }
            break;
         }

        int digit = num % 10;
        num = num/10;

        if(i % 2 == 1)
        {
            digit = digit * 2;
            if(digit > 9)
            {
                digit = sumDigits(digit);
            }
            multiplied += digit;
        }
        else if(i % 2 == 0){
            if(digit > 9)
            {
                digit = sumDigits(digit);
            }
            notMultiplied += digit;
        }


    }
    int sum = mutiplied + notMultiplied;
    if(sum % 10 == 0){

    }
    else{
        printf("INVALID");
    }
}

File: sentimental-credit/credit.py
# TODO
from cs50 import get_int

def main():
    num = get_int("Number: ")
    noOfDigits, multiplied, notMultiplied, firstNumber, secondNumber, i = 0, 0, 0, 0, 0, 0
    while True:
        noOfDigits += 1
        if num < 10:
            firstNumber = num
            if i % 2 == 1:
                num = num * 2
                if num > 9:
                    num = sumDigits(num)
                multiplied += num
            elif i % 2 == 0:
                notMultiplied += num
            break
        if num < 100:
            secondNumber = num % 10
        digit = num % 10
        num = num // 10
        if i % 2 == 1:
            digit = digit *2
            if digit > 9:
                digit = sumDigits(digit)
            multilplied += 





def sumDigits(n):
    return ( n % 10) + ( n // 10)

main()